src/bootstrap/src/core/config/config.rs

```diff
@@ -254,7 +254,6 @@ pub struct Config {
     pub rust_debuginfo_level_std: DebuginfoLevel,
     pub rust_debuginfo_level_tools: DebuginfoLevel,
     pub rust_debuginfo_level_tests: DebuginfoLevel,
-    pub rust_split_debuginfo: SplitDebuginfo,
```

Comment:
This breaking change can easily become painful for developers who switch back and forth between old and new commits. Please add a FIXME comment (e.g., FIXME: deprecated field, remove this in Q3) here instead of removing it.

Reply:
Ah, I hadn't considered the "switching back and forth" case. Will do.